extensions/console-extension.crd.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We'll probably need API review on the CRD

Reply:
This also should be added to the manifests directory of the console-operator so it's created with the console CRD